operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -291,6 +291,22 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		}
 	}
 
+	if !rendering.MCOAEnabled(instance) {
```

Comment:
We could add a filter on the request and run this only when the MCO config is modified?

Reply:
Do we have an example on the code base similar? I'm not quite sure how to accomplish this or I'm misunderstanding something